test/Dialect/FIRRTL/lower-intrinsics.mlir

```diff
@@ -156,4 +156,14 @@ firrtl.circuit "Foo" {
     // CHECK-NEXT: firrtl.int.fpga_probe %clock, %data : !firrtl.uint<32>
     firrtl.int.generic "circt_fpga_probe"  %data, %clock : (!firrtl.uint<32>, !firrtl.clock) -> ()
   }
+
+  // CHECK-LABEL: firrtl.module private @DPIIntrinsicTest(in %clock: !firrtl.clock, in %enable: !firrtl.uint<1>, in %in1: !firrtl.uint<2>, in %in2: !firrtl.uint<2>)
+  firrtl.module private @DPIIntrinsicTest(in %clock : !firrtl.clock, in %enable : !firrtl.uint<1>, in %in1: !firrtl.uint<2>, in %in2: !firrtl.uint<2>) {
+    // CHECK-NEXT: %0 = firrtl.int.dpi.call "clocked_result"(%in1, %in2) clock %clock enable %enable : (!firrtl.uint<2>, !firrtl.uint<2>) -> !firrtl.uint<2>
+    %0 = firrtl.int.generic "circt_dpi_call" <isClocked: ui32 = 1, functionName: none = "clocked_result"> %clock, %enable, %in1, %in2 : (!firrtl.clock, !firrtl.uint<1>, !firrtl.uint<2>, !firrtl.uint<2>) -> !firrtl.uint<2>
+    // CHECK-NEXT: firrtl.int.dpi.call "cloked_void"(%in1, %in2) clock %clock enable %enable : (!firrtl.uint<2>, !firrtl.uint<2>) -> ()
+    firrtl.int.generic "circt_dpi_call" <isClocked: ui32 = 1, functionName: none = "cloked_void"> %clock, %enable, %in1, %in2 : (!firrtl.clock, !firrtl.uint<1>, !firrtl.uint<2>, !firrtl.uint<2>) -> ()
```

Comment:
More "cloked" (below too).  Not a big deal but if easy might as well make this "clocked".

Reply:
Thanks for catching this!